github/client.go

```diff
@@ -177,6 +188,72 @@ func (c *Client) Token(ctx context.Context, opts *tokenOptions) (*logical.Respon
 	return tokenRes, nil
 }
 
+func (c *Client) getInstallationID(config *Config) (int, error) {
```

Comment:
Could you please write tests for this new method? Thanks.

Reply:
I will notify you when I will finish

Reply:
@martinbaillie can you review again please?